app/components/announcements/announcement-card.tsx

```diff
@@ -0,0 +1,49 @@
+import { ViewStyle, Platform, View, Text, TextStyle } from "react-native"
```

Comment:
@planecore Note that you have to use our own `Text` component (imported from the `components` folder) and not the react native `Text` component. That what caused the text to be RTL and not LTR